docs/mevblocker/builders/fees/subscription-fees.md

```diff
@@ -22,10 +22,36 @@ Despite the fact that the fee is per block, it will not be paid with each block:
 
 :::
 
-We'll compute the value of the per-block won fee using publicly available data, in a format that is easy to verify (for example, as a Dune query). The process for [posting](fee-management-smart-contract#billing-and-subscription-fees) this value on-chain, as well as posting builder's dues and payment is documented below in the [smart contract section](fee-management-smart-contract).
+We'll compute the value of the per-block won fee using publicly available data, in a format that is easy to verify (for example, as a Dune query). The process for [posting](fee-management-smart-contract#billing-and-subscription-fees) this value on-chain, as well as posting builders' dues and payment is documented below in the [smart contract section](fee-management-smart-contract).
 
 Finally, for builders who won fewer than 1% of blocks during a given week, the fee will be adjusted so that these builders pay as if they won exactly 1% of blocks during the week. In practice, at the end of each week, these builders pay:
 
 $$ 1\% (total\ blocks\ mined\ during\ the\ previous\ week) * 20\% M_{t-1} $$
 
-The rationale for this adjustment is that connecting a builder to MEV blocker poses a risk because of the potential leaking of transactions to the public mempool, which should be evaluated against the benefit of faster inclusion of MEV blocker transactions. We believe the risk outweighs the benefit for builders who win less than 1% of blocks. Although we do not want to penalize builders who may temporarily fall below this threshold too harshly, we also want to discourage builders that pose risk to the MEV Blocker service, from connecting in the first place.
```

Comment:
@gogonimago are we sure we want to have all this in a single page? I feel like it just makes the page super long

Reply:
I think "Why do we charge a fee" shouldn't be its own page and should just be included in the top-line info on the fee itself. It's longer but it's not unbearable IMO